sdk/search/azure-resourcemanager-search/README.md

```diff
@@ -0,0 +1,85 @@
+# Azure Resource Manager Azure AI Search client library for Java
+
+Azure Resource Manager Azure AI Search client library for Java
+
+For documentation on how to use this package, please see [Azure Management Libraries for Java](https://aka.ms/azsdk/java/mgmt).
+
+## Getting started
+
+### Prerequisites
+
+- [Java Development Kit (JDK)][jdk] with version 8 or above
+- [Azure Subscription][azure_subscription]
+
+### Adding the package to your product
+
+[//]: # ({x-version-update-start;com.azure.resourcemanager:azure-resourcemanager-search;current})
+```xml
+<dependency>
+    <groupId>com.azure.resourcemanager</groupId>
+    <artifactId>azure-resourcemanager-search</artifactId>
+    <version>2.53.0-beta.1</version>
+</dependency>
+```
+[//]: # ({x-version-update-end})
+
+### Include the recommended packages
+
+Azure Management Libraries require a `TokenCredential` implementation for authentication and an `HttpClient` implementation for HTTP client.
+
+[Azure Identity][azure_identity] package and [Azure Core Netty HTTP][azure_core_http_netty] package provide the default implementation.
+
+### Authentication
+
+Microsoft Entra ID token authentication relies on the [credential class][azure_identity_credentials] from [Azure Identity][azure_identity] package.
+
+Azure subscription ID can be configured via `AZURE_SUBSCRIPTION_ID` environment variable.
+Azure tenant ID can be configured via `AZURE_TENANT_ID` environment variable.
+
+Assuming the use of the `DefaultAzureCredential` credential class, the client can be authenticated using the following code:
+
+```java
+AzureProfile profile = new AzureProfile(AzureEnvironment.AZURE);
+TokenCredential credential = new DefaultAzureCredentialBuilder()
+    .authorityHost(profile.getEnvironment().getActiveDirectoryEndpoint())
+    .build();
+AzureResourceManager azure = AzureResourceManager
+    .authenticate(credential, profile)
+    .withDefaultSubscription();
+```
```

Comment:
Originally it's done by code-snippet tool. Since the code is not accessible now, directly copy the content here.